specification/cognitiveservices/data-plane/LUIS/Authoring/stable/v2.0/examples/apps/SuccessfulDownloadApplicationQueryLogsAsyncRequest.json

```diff
@@ -0,0 +1,15 @@
+{
+  "parameters": {
```

Comment:
There is an optional flag includeResponse, you need to document

Reply:
this is only an example on the API, the API is in LUIS-Authoring.json and it contains that parameter.